app/domain/authentication/authn_oidc/authenticate_id_token/decode_and_verify_id_token.rb

```diff
@@ -42,7 +42,7 @@ def verify_decoded_id_token
           raise IdTokenVerifyFailed, e.inspect
         end
 
-        def certs
+        def fetch_certs
           @certs ||= @provider_certificate.fetch_certs(@provider_uri)
```

Comment:
This doesn't actually need to be memoized. Just set it using `@certs =` and then refer to it as `@certs` in the method below. That should hopefully remove the CC warnings.